src/chen.jl

```diff
@@ -5,19 +5,32 @@ using LinearAlgebra
     RaoBlackwellLedoitWolf(shrinkage)
 
 Rao-Blackwell theorem Ledoit-Wolf covariance estimator. The parameter
-`shrinkage` is either equal to `:auto` and optimal shrinkage is calculated,
+`shrinkage` is either equal to `:optimal` and optimal shrinkage is calculated,
 or it is a number between 0 and 1.
 """
 struct RaoBlackwellLedoitWolf{S<:Union{Symbol, Real}} <: CovarianceEstimator
     shrinkage::S
+    function RaoBlackwellLedoitWolf(s::T) where T<:Real
+        @assert 0 ≤ s ≤ 1 "Shrinkage value should be between 0 and 1"
+        new{T}(s)
+    end
+    function RaoBlackwellLedoitWolf(s::Symbol=:optimal)
+        @assert s ∈ [:optimal] "Shrinkage setting not supported"
+        new{Symbol}(s)
+    end
 end
 
-RaoBlackwellLedoitWolf() = RaoBlackwellLedoitWolf{Symbol}(:auto)
+function chenoptimalshrinkage(n, p, C)
```

Comment:
I think this function should actually be called `raoblackwellledoitwolfshrinkage` (to make a point that it is different from Oracle Approximating Shrinkage).

Reply:
sounds good, see latest commit